viewer/src/main/webapp/viewer-html/components/AttributeList.js

```diff
@@ -311,13 +311,13 @@ Ext.define ("viewer.components.AttributeList",{
     },
     loadAttributes: function(appLayer) {
         var me = this;
-        if(this.requestThresholdCounter){
+        if (this.requestThresholdCounter) {
             clearTimeout(this.requestThresholdCounter);
         }
-        if(this.grids.main){
+        if (this.grids.main) {
             this.grids.main.getView().setLoading("Bezig met laden...");
         }
-        this.requestThresholdCounter =  setTimeout(function(){
```

Comment:
Personally I do not like the space between function and brackets, is this a setting of the IDE or was this on purpose?